Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -1080,14 +1080,19 @@ ImageEditor& MainWidget::create_new_editor(NonnullRefPtr<Image> image)
         auto const& image_size = current_image_editor()->image().size();
         auto image_rectangle = Gfx::IntRect { 0, 0, image_size.width(), image_size.height() };
         if (image_rectangle.contains(mouse_position)) {
-            m_statusbar->set_override_text(mouse_position.to_deprecated_string());
+            update_status_bar(current_image_editor()->appended_status_info());
             m_histogram_widget->set_color_at_mouseposition(current_image_editor()->image().color_at(mouse_position));
             m_vectorscope_widget->set_color_at_mouseposition(current_image_editor()->image().color_at(mouse_position));
         } else {
             m_statusbar->set_override_text({});
             m_histogram_widget->set_color_at_mouseposition(Color::Transparent);
             m_vectorscope_widget->set_color_at_mouseposition(Color::Transparent);
         }
```

Comment:
Same here, again :^)

Reply:
Unclear to me what you mean here now.
The update_status_bar will check if the appended text is empty or not and then update the text.
Is more optimal to check if appended status info is empty here and avoid that function call?

Reply:
Looking at it more closely, this situation is a little different so it's fine here